xpdacq/beamtimeSetup.py

```diff
@@ -33,7 +33,7 @@
 
 
 def _start_beamtime(PI_last, saf_num, experimenters=[],
-                    wavelength=None):
+                    wavelength=None,test=False):
```

Comment:
Can you please put a space between these two kwargs? (PEP8)

Reply:
:+1: